python/cudaq/kernel/utils.py

```diff
@@ -256,6 +276,29 @@ def emitFatalErrorOverride(msg):
     )
 
 
+def pyInstanceFromName(name: str):
+    if name == 'bool':
+        return bool(False)
+    if name == 'int':
+        return int(0)
+    if name in ['numpy.int32', 'np.int32']:
+        return np.int32(0)
+    if name in ['numpy.int64', 'np.int64']:
+        return np.int64(0)
+    if name == 'int':
+        return int(0)
+    if name == 'float':
+        return float(0.0)
+    if name == 'complex':
+        return 0j
+    if name == 'pauli_word':
+        return pauli_word('')
+    if name in ['numpy.complex128', 'np.complex128']:
+        return np.complex128(0.0)
+    if name in ['numpy.complex64', 'np.complex64']:
+        return np.complex64(0.0)
+
```

Comment:
I wonder whether we should emit an error if `name` was not handled.

Reply:
it currently just returns none and the caller emits errors depending on the context